# Polish translation of Update Manager.
# This file is distributed under the same license as the update-manager package.
# Copyright (c) 2006 Sebastian Heinlein
#               2007 Canonical
# Polish translation by:
# Zygmunt Krynicki <zkrynicki@gmail.com>, 2005-2006
# Tomasz Dominikowski <dominikowski@gmail.com>, 2006-2008
# Wiktor Wandachowicz <siryes@gmail.com>, 2008
#
# Nazewnictwo i spójność tłumaczeń programów apt, aptitude, synaptic i innych:
# http://wiki.debian.org/PolishL10N/PackageInstallers
msgid ""
msgstr ""
"Project-Id-Version: update-manager 0.87\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-10-18 03:11+0000\n"
"PO-Revision-Date: 2011-07-07 17:31+0000\n"
"Last-Translator: GTriderXC <gtriderxc@yahoo.com>\n"
"Language-Team: Polish <ubuntu-pl-l10n@lists.ubuntu.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
"|| n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2011-10-18 14:52+0000\n"
"X-Generator: Launchpad (build 14157)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:416 ../UpdateManager/Core/utils.py:416
msgid "0 kB"
msgstr ""

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:419 ../UpdateManager/Core/utils.py:419
msgid "1 kB"
msgstr ""

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:422 ../UpdateManager/Core/utils.py:422
#, python-format
msgid "%.0f kB"
msgstr ""

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:425 ../UpdateManager/Core/utils.py:425
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:210 ../DistUpgrade/distro.py:438
#, python-format
msgid "Server for %s"
msgstr "Serwer dla kraju: %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:228 ../DistUpgrade/distro.py:234
#: ../DistUpgrade/distro.py:250
msgid "Main server"
msgstr "Główny serwer"

#: ../DistUpgrade/distro.py:254
msgid "Custom servers"
msgstr "Inne serwery"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:141
msgid "Could not calculate sources.list entry"
msgstr "Nie można obliczyć wpisu w pliku w sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:244
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Nie można znaleźć żadnych plików z pakietami, może to nie jest płyta z "
"Ubuntu lub to niepoprawna architektura?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:287
msgid "Failed to add the CD"
msgstr "Nie można dodać napędu CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:288
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Wystąpił błąd podczas dodawania płyty CD, aktualizacja systemu zostanie "
"przerwana. Proszę zgłosić to jako błąd, jeśli jest to oficjalna płyta "
"Ubuntu.\n"
"\n"
"Treść błędu jest następująca:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:138
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Usuwanie uszkodzonego pakietu"
msgstr[1] "Usuwanie uszkodzonych pakietów"
msgstr[2] "Usuwanie uszkodzonych pakietów"

#: ../DistUpgrade/DistUpgradeCache.py:141
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Pakiet \"%s\" znajduje się w niezgodnym stanie i musi zostać zainstalowany "
"ponownie, ale nie odnaleziono żadnego archiwum tego pakietu. Usunąć pakiet "
"teraz aby kontynuować?"
msgstr[1] ""
"Pakiety \"%s\" znajdują się w niezgodnym stanie i muszą zostać zainstalowane "
"ponownie, ale nie odnaleziono żadnego archiwum tych pakietów. Usunąć pakiety "
"teraz aby kontynuować?"
msgstr[2] ""
"Pakiety \"%s\" znajdują się w niezgodnym stanie i muszą zostać zainstalowane "
"ponownie, ale nie odnaleziono żadnego archiwum tych pakietów. Usunąć pakiety "
"teraz aby kontynuować?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:239
msgid "The server may be overloaded"
msgstr "Serwer może być przeciążony"

#: ../DistUpgrade/DistUpgradeCache.py:352
msgid "Broken packages"
msgstr "Uszkodzone pakiety"

#: ../DistUpgrade/DistUpgradeCache.py:353
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"System zawiera uszkodzone pakiety, które nie mogły zostać naprawione. Przed "
"kontynuowaniem należy je naprawić używając programu Synaptic lub apt-get."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:648
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Wystąpił nierozwiązywalny problem podczas obliczania aktualizacji:\n"
"%s\n"
"\n"
" Może to być spowodowane przez:\n"
" * Aktualizowanie do testowej wersji Ubuntu\n"
" * Użytkowanie bieżącej testowej wersji Ubuntu\n"
" * Użytkowanie nieoficjalnych pakietów spoza repozytoriów Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:658
msgid "This is most likely a transient problem, please try again later."
msgstr "Wystąpiły przejściowe trudności. Proszę spróbować później."

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Jeśli żadna z wymienionych pozycji nie może zostać poprawnie zastosowana, "
"należy zgłosić błąd, wpisując w terminalu polecenie: \"ubuntu-bug update-"
"manager\""

#: ../DistUpgrade/DistUpgradeCache.py:666
#: ../UpdateManager/UpdateManager.py:1023
msgid "Could not calculate the upgrade"
msgstr "Nie można przetworzyć aktualizacji"

#: ../DistUpgrade/DistUpgradeCache.py:717
msgid "Error authenticating some packages"
msgstr "Błąd podczas weryfikacji autentyczności niektórych pakietów"

#: ../DistUpgrade/DistUpgradeCache.py:718
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Nie można było uwierzytelnić niektórych pakietów. Może to oznaczać chwilowy "
"błąd sieci. Można spróbować ponownie później. Poniżej znajduje się lista "
"nieuwierzytelnionych pakietów."

#: ../DistUpgrade/DistUpgradeCache.py:738
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Pakiet \"%s\" jest oznaczony do usunięcia, lecz znajduje się na liście "
"pakietów, których nie należy usuwać."

#: ../DistUpgrade/DistUpgradeCache.py:742
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Niezbędny pakiet \"%s\" jest oznaczony do usunięcia."

#: ../DistUpgrade/DistUpgradeCache.py:751
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Próba instalacji wersji \"%s\", znajdującej się na czarnej liście"

#: ../DistUpgrade/DistUpgradeCache.py:869
#, python-format
msgid "Can't install '%s'"
msgstr "Nie można zainstalować \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:870
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Instalacja wymaganych pakietów nie powiodła się. Proszę zgłosić ten błąd, "
"wpisując w terminalu polecenie: ubuntu-bug update-manager"

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:881
msgid "Can't guess meta-package"
msgstr "Nie można określić meta-pakietu"

#: ../DistUpgrade/DistUpgradeCache.py:882
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"System nie zawiera pakietów ubuntu-desktop, kubuntu-desktop, xubuntu-desktop "
"lub edubuntu-desktop i nie jest możliwe wykrycie używanej wersji Ubuntu.\n"
" Proszę zainstalować jeden z tych pakietów używając programu Synaptic lub "
"apt-get przed kontynuowaniem."

#: ../DistUpgrade/DistUpgradeController.py:100
msgid "Reading cache"
msgstr "Odczytywanie pamięci podręcznej"

#: ../DistUpgrade/DistUpgradeController.py:205
msgid "Unable to get exclusive lock"
msgstr "Nie można uzyskać blokady na wyłączność"

#: ../DistUpgrade/DistUpgradeController.py:206
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"To zazwyczaj oznacza, że inna aplikacja do zarządzania pakietami (jak apt-"
"get lub aptitude) jest już uruchomiona. Należy najpierw zamknąć tę aplikację."

#: ../DistUpgrade/DistUpgradeController.py:239
msgid "Upgrading over remote connection not supported"
msgstr "Aktualizacja poprzez połączenie zdalnie nie jest obsługiwana"

#: ../DistUpgrade/DistUpgradeController.py:240
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Uruchomiłeś aktualizację ze zdalnym połączeniem powłoki ssh z interfejsem, "
"który nie posiada takiego wsparcia. Spróbuj ponownie z 'do-release-"
"upgrade'.\n"
"\n"
"Aktualizacja będzie teraz przerwana. Spróbuj ponownie bez powłoki ssh."

#: ../DistUpgrade/DistUpgradeController.py:254
msgid "Continue running under SSH?"
msgstr "Kontynuować połączenie SSH?"

#: ../DistUpgrade/DistUpgradeController.py:255
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Uruchomiona sesja działa z powłoki ssh. Obecnie nie zaleca się dokonywania "
"uaktualnień z poziomu ssh ponieważ w przypadku błędu przywracanie systemu "
"może być trudniejsze.\n"
"\n"
"Jeżeli będziesz kontynuować, rozpoczęty zostanie dodatkowy demon w porcie "
"'%s'.\n"
"Czy chcesz  kontynuować?"

#: ../DistUpgrade/DistUpgradeController.py:269
msgid "Starting additional sshd"
msgstr "Uruchamianie dodatkowej usługi ssh"

#: ../DistUpgrade/DistUpgradeController.py:270
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Dodatkowy demon sshd zostanie uruchomiony na porcie '%s', aby ułatwić "
"odzyskiwanie w razie problemów z bieżącą usługą ssh. W przypadku "
"niepowodzenia z aktualną sesją ssh, wciąż można podłączyć się do sesji "
"dodatkowej.\n"

#: ../DistUpgrade/DistUpgradeController.py:278
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Jeśli używana jest zapora firewall, może okazać się konieczne tymczasowe "
"otwarcie tego portu. Czynność ta jest potencjalnie niebezpieczna, dlatego "
"nie zostanie wykonana automatycznie. Port może zostać otwarty między innymi "
"za pomocą:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:349
#: ../DistUpgrade/DistUpgradeController.py:394
msgid "Can not upgrade"
msgstr "Nie można zaktualizować"

#: ../DistUpgrade/DistUpgradeController.py:350
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr ""
"Aktualizacja z wersji \\\"%s\\\" do \\\"%s\\\" nie jest możliwa przy użyciu "
"tego narzędzia."

#: ../DistUpgrade/DistUpgradeController.py:359
msgid "Sandbox setup failed"
msgstr "Ustawienie trybu testowego nie powiodło się"

#: ../DistUpgrade/DistUpgradeController.py:360
msgid "It was not possible to create the sandbox environment."
msgstr "Utworzenie testowego środowiska nie było możliwe"

#: ../DistUpgrade/DistUpgradeController.py:366
msgid "Sandbox mode"
msgstr "Tryb testowy"

#: ../DistUpgrade/DistUpgradeController.py:367
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Aktualizacja działa w trybie testowym. Wszystkie zmiany są zapisywane do "
"\"%s\" i zostaną utracone po następnym uruchomieniu komputera.\n"
"\n"
"Do następnego uruchomienia komputera *wszystkie* zmiany wprowadzane do "
"katalogów systemowych zostaną utracone."

#: ../DistUpgrade/DistUpgradeController.py:395
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Instalacja python jest uszkodzona. Proszę naprawić dowiązanie do "
"\"/usr/bin/python\"."

#: ../DistUpgrade/DistUpgradeController.py:421
msgid "Package 'debsig-verify' is installed"
msgstr "Pakiet \"debsig-verify\" jest zainstalowany"

#: ../DistUpgrade/DistUpgradeController.py:422
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Aktualizacja nie może być kontynuowana, podczas gdy ten pakiet jest "
"zainstalowany.\n"
"Proszę najpierw usunąć go używając programu Synaptic lub \"apt-get remove "
"debsig-verify\", a następnie uruchomić ponownie aktualizację\"."

#: ../DistUpgrade/DistUpgradeController.py:432
msgid "Include latest updates from the Internet?"
msgstr "Dołączyć najnowsze aktualizacje z Internetu?"

#: ../DistUpgrade/DistUpgradeController.py:433
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"System aktualizacji może połączyć się z internetem, aby automatycznie pobrać "
"i zainstalować najnowsze aktualizacje. Jeśli masz połączenie sieciowe, jest "
"to opcja zalecana.\n"
"\n"
"Aktualizacja zabierze trochę czasu, ale gdy zostanie zakończona, system "
"będzie w pełni aktualny. Można tego zaniechać, jednak zalecana jest "
"instalacja najnowszych aktualizacji.\n"
"Jeśli teraz odpowiesz \"NIE\", sieć nie zostanie użyta."

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid "disabled on upgrade to %s"
msgstr "wyłączony podczas aktualizacji do %s"

#: ../DistUpgrade/DistUpgradeController.py:680
msgid "No valid mirror found"
msgstr "Nie odnaleziono poprawnego serwera lustrzanego"

#: ../DistUpgrade/DistUpgradeController.py:681
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Podczas przetwarzania informacji o repozytoriach nie odnaleziono wpisu "
"serwera lustrzanego do aktualizacji. Może się tak zdarzyć, gdy używamy "
"wewnętrznego serwera lustrzanego lub jeśli informacje o serwerach są "
"nieaktualne.\n"
"\n"
"Przepisać plik \"sources.list\" mimo to? Wybranie \"Tak\" oznacza "
"aktualizację wszystkich wpisów \"%s\" do \"%s\".\n"
"Wybranie \"Nie\" anuluje aktualizację."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:701
msgid "Generate default sources?"
msgstr "Wygenerować domyślne źródła?"

#: ../DistUpgrade/DistUpgradeController.py:702
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Podczas przetwarzania informacji o repozytoriach nie odnaleziono poprawnego "
"wpisu dla \"%s\".\n"
"\n"
"Dodać domyślne wpisy dla \"%s\"? Wybranie \"Nie\" przerwie aktualizację."

#: ../DistUpgrade/DistUpgradeController.py:737
msgid "Repository information invalid"
msgstr "Błędne informacje o repozytoriach"

#: ../DistUpgrade/DistUpgradeController.py:738
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug using the command 'ubuntu-bug update-manager' in a "
"terminal."
msgstr ""
"Aktualizacja informacji o zawartości repozytoriów nie powiodła się z powodu "
"błędów pliku. Proszę zgłosić ten błąd, wpisując w terminalu polecenie: "
"ubuntu-bug update-manager"

#: ../DistUpgrade/DistUpgradeController.py:746
msgid "Third party sources disabled"
msgstr "Źródła niezależnych dostawców zostały wyłączone"

#: ../DistUpgrade/DistUpgradeController.py:747
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Niektóre wpisy niezależnych dostawców w pliku \"sources.list\" zostały "
"wyłączone. Można ponownie je włączyć po aktualizacji używając narzędzia "
"\"software-properties\" lub programu Synaptic."

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakiet w niezgodnym stanie"
msgstr[1] "Pakiety w niezgodnym stanie"
msgstr[2] "Pakiety w niezgodnym stanie"

#: ../DistUpgrade/DistUpgradeController.py:790
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Pakiet \"%s\" znajduje się w niezgodnym stanie i musi zostać zainstalowany "
"ponownie, ale nie odnaleziono żadnego archiwum tego pakietu. Proszę "
"zainstalować pakiet ręcznie lub usunąć go z systemu."
msgstr[1] ""
"Pakiety \"%s\" znajdują się w niezgodnym stanie i muszą zostać zainstalowane "
"ponownie, ale nie odnaleziono żadnego archiwum tych pakietów. Proszę "
"zainstalować pakiety ręcznie lub usunąć je z systemu."
msgstr[2] ""
"Pakiety \"%s\" znajdują się w niezgodnym stanie i muszą zostać zainstalowane "
"ponownie, ale nie odnaleziono żadnego archiwum tych pakietów. Proszę "
"zainstalować pakiety ręcznie lub usunąć je z systemu."

#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Error during update"
msgstr "Błąd podczas aktualizacji"

#: ../DistUpgrade/DistUpgradeController.py:839
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Wystąpił problem podczas aktualizacji. Zazwyczaj wynika to z problemów z "
"siecią, proszę sprawdzić połączenie sieciowe i spróbować ponownie."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:848
#: ../UpdateManager/UpdateManager.py:747
msgid "Not enough free disk space"
msgstr "Zbyt mało miejsca na dysku"

#: ../DistUpgrade/DistUpgradeController.py:849
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Aktualizacja została przerwana. Aktualizacja wymaga ogółem %s wolnego "
"miejsca na dysku \"%s\". Proszę uwolnić przynajmniej %s dodatkowego wolnego "
"miejsca na \"%s\". Należy opróżnić kosz i usunąć tymczasowe pakiety z "
"poprzednich instalacji używając polecenia \"sudo apt-get clean\"."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:878
#: ../DistUpgrade/DistUpgradeController.py:1630
msgid "Calculating the changes"
msgstr "Przetwarzanie zmian"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:910
msgid "Do you want to start the upgrade?"
msgstr "Rozpocząć aktualizację?"

#: ../DistUpgrade/DistUpgradeController.py:976
msgid "Upgrade canceled"
msgstr "Aktualizacja anulowana"

#: ../DistUpgrade/DistUpgradeController.py:977
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Aktualizacja zostanie teraz anulowana, a system przywrócony do pierwotnego "
"stanu. Można ponowić próbę aktualizacji wersji później."

#: ../DistUpgrade/DistUpgradeController.py:983
#: ../DistUpgrade/DistUpgradeController.py:1109
msgid "Could not download the upgrades"
msgstr "Pobranie aktualizacji było niemożliwe"

#: ../DistUpgrade/DistUpgradeController.py:984
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1060
#: ../DistUpgrade/DistUpgradeController.py:1097
#: ../DistUpgrade/DistUpgradeController.py:1202
msgid "Error during commit"
msgstr "Błąd podczas zatwierdzania"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1062
#: ../DistUpgrade/DistUpgradeController.py:1099
#: ../DistUpgrade/DistUpgradeController.py:1241
msgid "Restoring original system state"
msgstr "Przywracanie pierwotnego stanu systemu"

#: ../DistUpgrade/DistUpgradeController.py:1063
#: ../DistUpgrade/DistUpgradeController.py:1078
#: ../DistUpgrade/DistUpgradeController.py:1100
msgid "Could not install the upgrades"
msgstr "Instalacja aktualizacji zakończyła się niepowodzeniem."

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1068
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Aktualizacja została przerwana. System może znajdować się w stanie "
"nienadającym się do użytku. Zostanie teraz uruchomione odzyskiwanie (dpkg --"
"configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1073
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug using the command 'ubuntu-bug update-manager' in a "
"terminal and include the files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Proszę zgłosić ten błąd, wpisując w terminalu polecenie: \"ubuntu-bug update-"
"manager\" oraz załączyć pliki z katalogu /var/log/dist-upgrade/ \n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1110
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Aktualizacja została przerwana. Proszę sprawdzić połączenie internetowe lub "
"nośnik instalacyjny i spróbować ponownie. "

#: ../DistUpgrade/DistUpgradeController.py:1190
msgid "Remove obsolete packages?"
msgstr "Usunąć przestarzałe pakiety?"

#: ../DistUpgrade/DistUpgradeController.py:1191
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Zachowaj"

#: ../DistUpgrade/DistUpgradeController.py:1191
msgid "_Remove"
msgstr "_Usuń"

#: ../DistUpgrade/DistUpgradeController.py:1203
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "Wystąpił problem podczas czyszczenia. Więcej informacji poniżej. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "Required depends is not installed"
msgstr "Wymagana zależność nie jest zainstalowana."

#: ../DistUpgrade/DistUpgradeController.py:1280
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Wymagana zależność \"%s\" nie jest zainstalowana. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1527
#: ../DistUpgrade/DistUpgradeController.py:1597
msgid "Checking package manager"
msgstr "Sprawdzanie menedżera pakietów"

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Preparing the upgrade failed"
msgstr "Przygotowanie aktualizacji nie powiodło się"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"Preparing the system for the upgrade failed. Please report this using the "
"command 'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"Przygotowanie systemu do aktualizacji zakończyło się niepowodzeniem. Proszę "
"zgłosić ten błąd, wpisując w terminalu polecenie: \"ubuntu-bug update-"
"manager\" i załączyć do raportu pliki z katalogu /var/log/dist-upgrade/"

#: ../DistUpgrade/DistUpgradeController.py:1549
msgid "Getting upgrade prerequisites failed"
msgstr "Pobranie elementów wymaganych do aktualizacji nie powiodło się"

#: ../DistUpgrade/DistUpgradeController.py:1550
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug using the command 'ubuntu-bug update-manager' in "
"a terminal and include the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Niezbędne do aktualizacji wymagania systemowe nie zostały spełnione. Proces "
"zostanie przerwany, a system przywrócony do stanu sprzed rozpoczęcia "
"operacji.\n"
"\n"
"Proszę zgłosić ten błąd wpisując w terminalu polecenie: \"ubuntu-bug update-"
"manager\" oraz załączyć do raportu pliki z katalogu /var/log/dist-upgrade/"

#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Updating repository information"
msgstr "Aktualizowanie informacji o repozytoriach"

#: ../DistUpgrade/DistUpgradeController.py:1588
msgid "Failed to add the cdrom"
msgstr "Dodanie CD-ROM nieudane"

#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Sorry, adding the cdrom was not successful."
msgstr "Dodanie napędu CD-ROM zakończone niepowodzeniem"

#: ../DistUpgrade/DistUpgradeController.py:1617
msgid "Invalid package information"
msgstr "Błędne informacje o pakietach"

#: ../DistUpgrade/DistUpgradeController.py:1618
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"Po aktualizacji informacji o pakiecie, nie można odnaleźć niezbędnej paczki "
"'%s'.\n"
"To jest poważny błąd, który powinien zostać zgłoszony, poprzez wpisanie w "
"terminalu: \"ubuntu-bug update-manager\" oraz załączenie do raportu plików z "
"katalogu /var/log/dist-upgrade/"

#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1689
msgid "Fetching"
msgstr "Pobieranie"

#: ../DistUpgrade/DistUpgradeController.py:1642
#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Upgrading"
msgstr "Aktualizacja w toku"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1647
#: ../DistUpgrade/DistUpgradeController.py:1695
#: ../DistUpgrade/DistUpgradeController.py:1702
#: ../DistUpgrade/DistUpgradeController.py:1713
msgid "Upgrade complete"
msgstr "Aktualizacja ukończona"

#: ../DistUpgrade/DistUpgradeController.py:1648
#: ../DistUpgrade/DistUpgradeController.py:1696
#: ../DistUpgrade/DistUpgradeController.py:1703
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Aktualizacja została zakończona, lecz wystąpiły błędy podczas tego procesu."

#: ../DistUpgrade/DistUpgradeController.py:1655
msgid "Searching for obsolete software"
msgstr "Wyszukiwanie przestarzałego oprogramowania"

#: ../DistUpgrade/DistUpgradeController.py:1664
msgid "System upgrade is complete."
msgstr "Aktualizacja systemu zakończona powodzeniem."

#: ../DistUpgrade/DistUpgradeController.py:1714
msgid "The partial upgrade was completed."
msgstr "Częściowa aktualizacja została ukończona"

#: ../DistUpgrade/DistUpgradeQuirks.py:201
msgid "evms in use"
msgstr "EVMS w użyciu"

#: ../DistUpgrade/DistUpgradeQuirks.py:202
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Twój system używa managera woluminów EVMS w /proc/mounts. Oprogramowanie "
"EVMS nie jest więcej wspierane. W związku z tym należy je wyłączyć i "
"zaktualizować, a po jej zakończeniu włączyć go ponownie."

#: ../DistUpgrade/DistUpgradeQuirks.py:506
msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
msgstr ""
"Obsługa sprzętowa grafiki może nie być w pełni wspierana w tej wersji Ubuntu "
"11.04"

#: ../DistUpgrade/DistUpgradeQuirks.py:508
msgid ""
"The support in Ubuntu 11.04 for your intel graphics hardware is limited and "
"you may encounter problems after the upgrade. Do you want to continue with "
"the upgrade?"
msgstr ""
"Obsługa sprzętowa dla grafiki urządzeń Intel jest w wersji Ubuntu 11.04 "
"ograniczona. Po aktualizacji mogą pojawić się problemy. Czy kontynuować mimo "
"wszystko?"

#: ../DistUpgrade/DistUpgradeQuirks.py:528
#: ../DistUpgrade/DistUpgradeQuirks.py:556
#: ../DistUpgrade/DistUpgradeQuirks.py:583
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Aktualizacja może ograniczyć wydajność systemu dla efektów pulpitu, gier "
"oraz innych programów obciążających zasoby graficzne."

#: ../DistUpgrade/DistUpgradeQuirks.py:532
#: ../DistUpgrade/DistUpgradeQuirks.py:560
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Komputer korzysta ze sterownika grafiki NVIDIA \"nvidia\". Brak wersji "
"sterownika z którą współpracowałby ten sprzęt w Ubuntu 10.04 LTS.\n"
"\n"
"Kontynuować?"

#: ../DistUpgrade/DistUpgradeQuirks.py:587
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Komputer korzysta ze sterownika grafiki AMD \"fglrx\". Brak wersji "
"sterownika z którą współpracowałby ten sprzęt w Ubuntu 10.04 LTS.\n"
"\n"
"Kontynuować?"

#: ../DistUpgrade/DistUpgradeQuirks.py:617
msgid "No i686 CPU"
msgstr "Brak procesora o architekturze i686"

#: ../DistUpgrade/DistUpgradeQuirks.py:618
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Twój system używa procesora o architekturze i586 lub twój procesor nie "
"posiada rozszerzenia 'cmov'. Wszystkie pakiety zostały zbudowane w oparciu o "
"optymalizacje wymagające minimum architektury i686. Na tym sprzęcie "
"niemożliwa jest aktualizacja twojego systemu do nowego wydania Ubuntu."

#: ../DistUpgrade/DistUpgradeQuirks.py:654
msgid "No ARMv6 CPU"
msgstr "Brak procesora ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:655
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ten system korzysta z procesora ARM o architekturze starszej niż ARMv6. "
"Optymalizacje wszystkich pakietów w wydaniu Karmic wymagają minimalnie "
"architektury ARMv6. Nie ma możliwości aktualizacji systemu do nowego wydania "
"na tym sprzęcie."

#: ../DistUpgrade/DistUpgradeQuirks.py:675
msgid "No init available"
msgstr "Brak dostępnego procesu init"

#: ../DistUpgrade/DistUpgradeQuirks.py:676
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Wygląda na to, że system jest uruchomiony w środowisku wirtualnym bez "
"procesu init, np. Linux-VServer. System Ubuntu 10.04 LTS nie może pracować z "
"tym typem środowiska, wymagając najpierw aktualizacji konfiguracji maszyny "
"wirtualnej."

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Sandbox upgrade using aufs"
msgstr "Aktualizacja testowa używając aufs"

#: ../DistUpgrade/DistUpgradeMain.py:47
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Użycie podanej ścieżki do wyszukiwania CD-ROMu z pakietami aktualizacyjnymi"

#: ../DistUpgrade/DistUpgradeMain.py:53
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Użycie nakładki graficznej. Obecnie dostępne: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:56
msgid "*DEPRECATED* this option will be ignored"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Wykonaj tylko częściową aktualizację (bez nadpisywania sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid "Disable GNU screen support"
msgstr "Wyłącz wsparcie ekranu GNU"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Set datadir"
msgstr "Ustaw ścieżkę do danych"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:189
#: ../UpdateManager/DistUpgradeFetcherKDE.py:147
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Proszę włożyć \"%s\" do napędu \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Pobieranie zakończone"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Pobieranie %li pliku z %li z prędkością %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:295
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:301
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:361
#, python-format
msgid "About %s remaining"
msgstr "Pozostało około %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "Pobieranie pliku %li z %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Zatwierdzanie zmian"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:211
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "problemy z zależnościami - pozostawiony nieskonfigurowany"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "Nie można było zainstalować \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Aktualizacja będzie kontynuowana, ale pakiet \"%s\" może znajdować się w "
"stanie nienadającym się do pracy. Proszę rozważyć zgłoszenie raportu błędu "
"na ten temat."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:234
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Zastąpić własny plik konfiguracji\n"
"\"%s\"?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Zostaną utracone wszystkie zmiany wprowadzone w tym pliku konfiguracyjnym, "
"jeśli zostanie wybrana jego zamiana na nowszą wersję."

#: ../DistUpgrade/DistUpgradeViewGtk.py:250
#: ../DistUpgrade/DistUpgradeViewGtk3.py:253
#: ../DistUpgrade/DistUpgradeViewKDE.py:313
msgid "The 'diff' command was not found"
msgstr "Polecenie \"diff\" nie zostało odnalezione"

#: ../DistUpgrade/DistUpgradeViewGtk.py:463
#: ../DistUpgrade/DistUpgradeViewGtk3.py:469
#: ../DistUpgrade/DistUpgradeViewText.py:90
msgid "A fatal error occurred"
msgstr "Wystąpił błąd krytyczny"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:470
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Proszę wypełnić raport błędu (jeśli nie zostało to jeszcze zrobione) i "
"dołączyć w nim pliki var/log/dist-upgrade/main.log i /var/log/dist-"
"upgrade/apt.log. Aktualizacja została przerwana.\n"
"Pierwotny plik sources.list został zapisany w "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:481
#: ../DistUpgrade/DistUpgradeViewGtk3.py:487
msgid "Ctrl-c pressed"
msgstr "Wciśnięto Ctrl-C"

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:488
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Działanie zostanie przerwane i może pozostawić system w uszkodzonym stanie. "
"Na pewno chcesz to zrobić?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewGtk3.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Aby zapobiec utracie danych proszę zamknąć wszystkie otwarte programy i "
"dokumenty."

#: ../DistUpgrade/DistUpgradeViewGtk.py:644
#: ../DistUpgrade/DistUpgradeViewGtk3.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Już nie jest wspierane przez firmę Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Zainstaluj poprzednią wersję (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:637
#, python-format
msgid "Remove (%s)"
msgstr "Usuń (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr "Już nie jest potrzebne (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:639
#, python-format
msgid "Install (%s)"
msgstr "Zainstaluj (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr "Zaktualizuj (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:149
msgid "Media Change"
msgstr "Zmiana nośnika"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "Show Difference >>>"
msgstr "Wyświetl różnicę >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:328
msgid "<<< Hide Difference"
msgstr "<<< Ukryj różnicę"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "Error"
msgstr "Wystąpił błąd"

#: ../DistUpgrade/DistUpgradeViewKDE.py:555
msgid "&Cancel"
msgstr "&Anuluj"

#: ../DistUpgrade/DistUpgradeViewKDE.py:557
#: ../DistUpgrade/DistUpgradeViewKDE.py:799
msgid "&Close"
msgstr "&Zakończ"

#: ../DistUpgrade/DistUpgradeViewKDE.py:603
msgid "Show Terminal >>>"
msgstr "Wyświetl terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:606
msgid "<<< Hide Terminal"
msgstr "<<< Ukryj terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:686
msgid "Information"
msgstr "Informacje"

#: ../DistUpgrade/DistUpgradeViewKDE.py:737
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Szczegóły"

#: ../DistUpgrade/DistUpgradeViewKDE.py:763
#, python-format
msgid "No longer supported %s"
msgstr "Już nie jest obsługiwane (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#, python-format
msgid "Remove %s"
msgstr "Usuń %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#: ../DistUpgrade/DistUpgradeViewText.py:179
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Usuń (było automatycznie zainstalowane) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:769
#, python-format
msgid "Install %s"
msgstr "Instaluj %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:771
#, python-format
msgid "Upgrade %s"
msgstr "Aktualizuj %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
#: ../DistUpgrade/DistUpgradeViewText.py:227
msgid "Restart required"
msgstr "Wymagane ponowne uruchomienie komputera"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Uruchom ponownie komputer w celu zakończenia aktualizacji</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:798 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid "_Restart Now"
msgstr "_Uruchom ponownie"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:816
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Czy przerwać bieżącą aktualizację ?</big></b>\n"
"\n"
"System może utracić stabilność jeśli przerwiesz aktualizację. Wskazane jest "
"wznowienie aktualizacji."

#: ../DistUpgrade/DistUpgradeViewKDE.py:820
msgid "Cancel Upgrade?"
msgstr "Anulować aktualizację?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dzień"
msgstr[1] "%li dni"
msgstr[2] "%li dni"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li godzina"
msgstr[1] "%li godziny"
msgstr[2] "%li godzin"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuta"
msgstr[1] "%li minuty"
msgstr[2] "%li minut"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekunda"
msgstr[1] "%li sekundy"
msgstr[2] "%li sekund"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:98
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:147
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Pobieranie potrwa około %s używając połączenia 1 Mbit DSL lub około %s "
"używając połączenia modemowego 56k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Pobieranie może potrwać około %s w przypadku tego łącza. "

#: ../DistUpgrade/DistUpgradeView.py:253 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Przygotowywanie do aktualizacji"

#: ../DistUpgrade/DistUpgradeView.py:254
msgid "Getting new software channels"
msgstr "Pobieranie nowych kanałów oprogramowania"

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Pobieranie nowych pakietów"

#: ../DistUpgrade/DistUpgradeView.py:256 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Instalowanie aktualizacji"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Czyszczenie"

#: ../DistUpgrade/DistUpgradeView.py:342
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d zainstalowany pakiet nie jest już obsługiwany przez firmę "
"Canonical. Nadal można uzyskać wsparcie od społeczności."
msgstr[1] ""
"%(amount)d zainstalowane pakiety nie są już obsługiwane przez firmę "
"Canonical. Nadal można uzyskać wsparcie od społeczności."
msgstr[2] ""
"%(amount)d zainstalowanych pakietów nie jest już obsługiwanych przez firmę "
"Canonical. Nadal można uzyskać wsparcie od społeczności."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:351
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakiet zostanie usunięty."
msgstr[1] "%d pakiety zostaną usunięte."
msgstr[2] "%d pakietów zostanie usuniętych."

#: ../DistUpgrade/DistUpgradeView.py:356
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nowy pakiet zostanie zainstalowany."
msgstr[1] "%d nowe pakiety zostaną zainstalowane."
msgstr[2] "%d nowych pakietów zostanie zainstalowanych."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakiet zostanie zaktualizowany."
msgstr[1] "%d pakiety zostaną zaktualizowane."
msgstr[2] "%d pakietów zostanie zaktualizowanych."

#: ../DistUpgrade/DistUpgradeView.py:367
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Konieczne pobranie w sumie %s. "

#: ../DistUpgrade/DistUpgradeView.py:371
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Pobieranie i instalowanie aktualizacji może potrwać kilka godzin. Gdy "
"pobieranie zostanie zakończone, procesu nie będzie można anulować."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:377 ../UpdateManager/UpdateManager.py:664
msgid "Your system is up-to-date"
msgstr "System jest w pełni zaktualizowany"

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "Brak dostępnych aktualizacji. Aktualizacja zostanie anulowana."

#: ../DistUpgrade/DistUpgradeView.py:391
msgid "Reboot required"
msgstr "Wymagane ponowne uruchomienie komputera"

#: ../DistUpgrade/DistUpgradeView.py:392
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Aktualizacja została ukończona i należy ponownie uruchomić komputer. "
"Uruchomić ponownie teraz?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid "Could not run the upgrade tool"
msgstr "Nie można było uruchomić narzędzia aktualizacji"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"Prawdopodobnie wystąpił błąd narzędzia aktualizacji. Proszę zgłosić ten "
"błąd, wpisując w terminalu polecenie: \"ubuntu-bug update-manager\""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool signature"
msgstr "Podpis narzędzia aktualizacji"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:214
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:214
msgid "Upgrade tool"
msgstr "Narzędzie aktualizacji"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Failed to fetch"
msgstr "Pobieranie nie powiodło się"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Pobieranie aktualizacji nie powiodło się. Może to oznaczać problem z siecią. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:253
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:253
msgid "Failed to extract"
msgstr "Rozpakowywanie nie powiodło się"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Rozpakowywanie aktualizacji nie powiodło się. Mógł wystąpić problem z siecią "
"lub z serwerem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:259
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:259
msgid "Verification failed"
msgstr "Weryfikacja nie powiodła się"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:260
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:260
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Weryfikacja aktualizacji nie powiodła się. Mógł wystąpić problem z siecią "
"lub z serwerem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:265
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:265
msgid "Authentication failed"
msgstr "Uwierzytelnienie nie powiodło się"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:266
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Uwierzytelnienie aktualizacji nie powiodło się. Mógł wystąpić problem z "
"siecią lub z serwerem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid "Can not run the upgrade"
msgstr "Nie można rozpocząć aktualizacji"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:281
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:281
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Taka sytuacja ma najczęściej miejsce w systemie gdzie katalog /tmp jest "
"zamontowany z opcją noexec. Proszę zamontować go ponownie bez opcji noexec i "
"uruchomić aktualizację ponownie."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:287
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:287
#, python-format
msgid "The error message is '%s'."
msgstr "Treść komunikatu błędu: \\\"%s\\\""

#: ../DistUpgrade/DistUpgradeViewText.py:91
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Proszę wypełnić raport błędu i dołączyć do niego pliki /var/log/dist-"
"upgrade/main.log i /var/log/dist-upgrade/apt.log. Aktualizacja została "
"przerwana.\n"
"Pierwotny plik sources.list został zapisany w "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:115
msgid "Aborting"
msgstr "Anulowanie"

#: ../DistUpgrade/DistUpgradeViewText.py:120
msgid "Demoted:\n"
msgstr "Zdegradowane:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:127
msgid "To continue please press [ENTER]"
msgstr "Aby kontynuować, naciśnij ENTER"

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
#: ../DistUpgrade/DistUpgradeViewText.py:200
msgid "Continue [yN] "
msgstr "Kontynuuj [tN] "

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
msgid "Details [d]"
msgstr "Szczegóły [s]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:159
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "y"
msgstr "t"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "d"
msgstr "s"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "No longer supported: %s\n"
msgstr "Już nie jest obsługiwane: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove: %s\n"
msgstr "Usuń: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "Install: %s\n"
msgstr "Instaluj: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Upgrade: %s\n"
msgstr "Aktualizuj: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid "Continue [Yn] "
msgstr "Kontynuować [Tn] "

#: ../DistUpgrade/DistUpgradeViewText.py:228
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Aby ukończyć aktualizację należy uruchomić ponownie komputer.\n"
"Po wybraniu \"t\" komputer zostanie uruchomiony ponownie."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Przerwać trwającą aktualizację?</big></b>\n"
"\n"
"System może stać się niezdatny do użytku jeśli aktualizacja zostanie "
"przerwana. Zalecane jest kontynuowanie aktualizacji."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Uruchom ponownie komputer w celu zakończenia aktualizacji</big></b>\n"
"\n"
"Proszę zapisać swoją pracę przed kontynuacją."

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Rozpocząć aktualizację?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 11.10</big></b>"
msgstr "<b><big>Aktualizacja Ubuntu do wersji 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Różnice pomiędzy plikami"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Aktualizacja dystrybucji"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Ponowne uruchamianie komputera"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Ustawianie nowych kanałów oprogramowania"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Terminal"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "_Przerwij aktualizację"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "K_ontynuuj"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "Zas_tąp"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "Zgłoś _błąd"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "_Wznów aktualizację"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "_Rozpocznij aktualizację"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:55
msgid "Please wait, this can take some time."
msgstr "Proszę czekać, to może chwilę potrwać."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:57
msgid "Update is complete"
msgstr "Aktualizacja została ukończona."

#: ../UpdateManager/DistUpgradeFetcher.py:108
#: ../UpdateManager/DistUpgradeFetcherKDE.py:102
msgid "Could not find the release notes"
msgstr "Nie można było odnaleźć informacji o wydaniu"

#: ../UpdateManager/DistUpgradeFetcher.py:109
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "The server may be overloaded. "
msgstr "Serwer może być przeciążony. "

#: ../UpdateManager/DistUpgradeFetcher.py:119
#: ../UpdateManager/DistUpgradeFetcherKDE.py:107
msgid "Could not download the release notes"
msgstr "Pobranie informacji o wydaniu było niemożliwe"

#: ../UpdateManager/DistUpgradeFetcher.py:120
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Please check your internet connection."
msgstr "Proszę sprawdzić połączenie sieciowe."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:61
#: ../UpdateManager/DistUpgradeFetcherKDE.py:84
msgid "Upgrade"
msgstr "Aktualizuj"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:88
#: ../data/gtkbuilder/UpdateManager.ui.h:13
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Informacje o wydaniu"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:127
#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
msgid "Downloading additional package files..."
msgstr "Pobieranie dodatkowych plików pakietów..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Plik %s z %s z prędkością %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
#, python-format
msgid "File %s of %s"
msgstr "Plik %s z %s"

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr "Otwórz odnośnik w przeglądarce"

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr "Skopiuj odnośnik do schowka"

#: ../UpdateManager/GtkProgress.py:159
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Pobieranie pliku %(current)li z %(total)li z prędkością %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:164
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Pobieranie pliku %(current)li z %(total)li"

#: ../UpdateManager/UpdateManager.py:105 ../do-release-upgrade:90
msgid "Your Ubuntu release is not supported anymore."
msgstr "Twoja wersja Ubuntu nie jest już wspierana."

#: ../UpdateManager/UpdateManager.py:106
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu Linux."
msgstr ""
"Nie otrzymasz więcej żadnych poprawek bezpieczeństwa i aktualizacji "
"krytycznych. Uaktualnij Ubuntu do późniejszej wersji."

#: ../UpdateManager/UpdateManager.py:114
msgid "Upgrade information"
msgstr "Informacje o aktualizacji"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:389
#, python-format
msgid "Version %s: \n"
msgstr "Wersja %s: \n"

#: ../UpdateManager/UpdateManager.py:445
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Nie wykryto połączenia sieciowego. Informacja changelog gotowa do pobrania."

#: ../UpdateManager/UpdateManager.py:453
msgid "Downloading list of changes..."
msgstr "Pobieranie listy zmian..."

#: ../UpdateManager/UpdateManager.py:497
msgid "_Deselect All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:503
msgid "Select _All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:552
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] "Wybrano %(count)s aktualizację."
msgstr[1] "Wybrano %(count)s aktualizacje."
msgstr[2] "Wybrano %(count)s aktualizacji."

#: ../UpdateManager/UpdateManager.py:556
#, python-format
msgid "%s will be downloaded."
msgstr "%s zostanie pobranych."

#: ../UpdateManager/UpdateManager.py:568
msgid "The update has already been downloaded, but not installed"
msgid_plural "The updates have already been downloaded, but not installed"
msgstr[0] "Aktualizacja została już pobrana, ale nie została zainstalowana"
msgstr[1] "Aktualizacje zostały już pobrane, ale nie zostały zainstalowane"
msgstr[2] "Aktualizacje zostały już pobrane, ale nie zostały zainstalowane"

#: ../UpdateManager/UpdateManager.py:573
msgid "There are no updates to install"
msgstr "Brak aktualizacji do zainstalowania"

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:579
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:586
msgid "Unknown download size."
msgstr "Nieznany rozmiar pobrania."

#: ../UpdateManager/UpdateManager.py:612
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""

#: ../UpdateManager/UpdateManager.py:618
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Informacje o pakiecie zostały zaktualizowane %(days_ago)s dni temu.\n"
"Naciśnij poniższy przycisk 'Sprawdź' aby sprawdzić najnowsze aktualizacje "
"oprogramowania."

#: ../UpdateManager/UpdateManager.py:623
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] ""
"Informacje o pakietach zostały ostatnio zaktualizowane %(days_ago)s dzień "
"temu."
msgstr[1] ""
"Informacje o pakietach zostały ostatnio zaktualizowane %(days_ago)s dni temu."
msgstr[2] ""
"Informacje o pakietach zostały ostatnio zaktualizowane %(days_ago)s dni temu."

#: ../UpdateManager/UpdateManager.py:627
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
"Informacje o pakietach zostały ostatnio zaktualizowane %(hours_ago)s godzinę "
"temu."
msgstr[1] ""
"Informacje o pakietach zostały ostatnio zaktualizowane %(hours_ago)s godziny "
"temu."
msgstr[2] ""
"Informacje o pakietach zostały ostatnio zaktualizowane %(hours_ago)s godzin "
"temu."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:632 ../UpdateManager/UpdateManager.py:634
#: ../UpdateManager/UpdateManager.py:636
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr ""

#: ../UpdateManager/UpdateManager.py:638
msgid "The package information was just updated."
msgstr ""

#: ../UpdateManager/UpdateManager.py:658
#: ../data/gtkbuilder/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Aktualizacje oprogramowania poprawiają błędy, eliminują luki bezpieczeństwa "
"i dostarczają nowe funkcje."

#: ../UpdateManager/UpdateManager.py:677
msgid "Software updates may be available for your computer."
msgstr "Dla tego komputera mogą być dostępne aktualizacje oprogramowania."

#: ../UpdateManager/UpdateManager.py:685
msgid "Welcome to Ubuntu"
msgstr "Witamy w Ubuntu"

#: ../UpdateManager/UpdateManager.py:687
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from Applications later."
msgstr ""

#: ../UpdateManager/UpdateManager.py:689
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from the Administration Menu later."
msgstr ""

#: ../UpdateManager/UpdateManager.py:692
msgid "Software updates are available for this computer"
msgstr "Dostępne są aktualizacje oprogramowania"

#: ../UpdateManager/UpdateManager.py:694
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from "
"Applications later."
msgstr ""
"Aby zainstalować aktualizacje później, należy ręcznie uruchomić Menedżera "
"aktualizacji z menu System."

#: ../UpdateManager/UpdateManager.py:696
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Można zainstalować je później, wybierając pozycję \"Menedżer aktualizacji\" "
"z menu Administracja."

#: ../UpdateManager/UpdateManager.py:748
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Aktualizacja wymaga %s wolnego miejsca na dysku \"%s\". Proszę uwolnić co "
"najmniej %s dodatkowego miejsca na \"%s\". Proszę opróżnić kosz i usunąć "
"tymczasowe pakiety z poprzednich instalacji za pomocą polecenia \"sudo apt-"
"get clean\"."

#: ../UpdateManager/UpdateManager.py:773
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Komputer musi zostać ponownie uruchomiony, aby ukończyć instalowanie "
"aktualizacji. Proszę zapisać bieżącą pracę przed kontynuowaniem."

#: ../UpdateManager/UpdateManager.py:837
msgid "Reading package information"
msgstr "Odczytywanie informacji o pakietach"

#: ../UpdateManager/UpdateManager.py:852
msgid "Connecting..."
msgstr "Nawiązywanie połączenia..."

#: ../UpdateManager/UpdateManager.py:869
msgid "You may not be able to check for updates or download new updates."
msgstr ""
"Sprawdzenie dostępności aktualizacji lub pobranie nowych aktualizacji może "
"się nie powieść."

#: ../UpdateManager/UpdateManager.py:994
msgid "Could not initialize the package information"
msgstr "Nie można zainicjować informacji o pakietach"

#: ../UpdateManager/UpdateManager.py:995
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Wystąpił niemożliwy do rozwiązania problem podczas inicjalizacji informacji "
"o pakietach.\n"
"\n"
"Proszę zgłosić ten błąd dla pakietu \\\"update-manager\\\" załączając "
"poniższą treść komunikatu błędu:\n"

#: ../UpdateManager/UpdateManager.py:1024
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Wystąpił niemożliwy do rozwiązania problem podczas obliczania aktualizacji.\n"
"\n"
"Proszę zgłosić ten błąd dla pakietu \\\"update-manager\\\" załączając "
"poniższą treść komunikatu błędu:"

#: ../UpdateManager/UpdateManager.py:1049
msgid " (New install)"
msgstr " (Nowa instalacja)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1056
#, python-format
msgid "(Size: %s)"
msgstr "(Rozmiar: %s)"

#: ../UpdateManager/UpdateManager.py:1058
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Z wersji %(old_version)s do %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1062
#, python-format
msgid "Version %s"
msgstr "Wersja %s"

#: ../UpdateManager/UpdateManager.py:1093 ../do-release-upgrade:102
msgid "Release upgrade not possible right now"
msgstr "Aktualizacja wydania jest teraz niemożliwa"

#: ../UpdateManager/UpdateManager.py:1094 ../do-release-upgrade:103
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Aktualizacja dystrybucji nie może zostać wykonana, proszę spróbować później. "
"Komunikat serwera: \"%s\""

#: ../UpdateManager/UpdateManager.py:1096 ../check-new-release-gtk:116
msgid "Downloading the release upgrade tool"
msgstr "Pobieranie narzędzia aktualizacji wydania"

#: ../UpdateManager/UpdateManager.py:1103
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Dostępne nowe wydanie Ubuntu: \"%s\"</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1142
msgid "Software index is broken"
msgstr "Spis oprogramowania jest uszkodzony"

#: ../UpdateManager/UpdateManager.py:1143
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Nie ma możliwości instalacji lub usunięcia jakiegokolwiek oprogramowania. "
"Proszę użyć programu Synaptic lub wydać polecenie \"sudo apt-get install -"
"f\" w terminalu, aby naprawić ten problem."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Sprawdź aktualizacje"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Instalacja wszystkich dostępnych aktualizacji"

#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr "Anuluj"

#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr "Zainstaluj"

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr "Tworzenie listy pakietów do uaktualnienia"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Zwykła aktualzacja nie mogła zostać obliczona, proszę uruchomić: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Może to być spowodowane przez:\n"
" * Niedokończony poprzedni proces aktualizacji\n"
" * Problemy z niektórymi pakietami oprogramowania\n"
" * Nieoficjalnymi pakietami spoza repozytorium Ubuntu\n"
" * Normalnymi zmianiami w testowej wersji Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr "Pobieranie dziennika zmian"

#: ../UpdateManager/Core/MyCache.py:128
#, python-format
msgid "Other updates (%s)"
msgstr "Inne aktualizacje (%s)"

#: ../UpdateManager/Core/MyCache.py:283
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""

#: ../UpdateManager/Core/MyCache.py:297
msgid "This update does not come from a source that supports changelogs."
msgstr "Ta aktualizacja nie pochodzi ze źródła wspierającego changelogs."

#: ../UpdateManager/Core/MyCache.py:303 ../UpdateManager/Core/MyCache.py:324
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Nie udało się pobrać listy zmian. \n"
"Proszę sprawdzić połączenie sieciowe."

#: ../UpdateManager/Core/MyCache.py:313
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Dziennik zmian jest niedostępny.\n"
"\n"
"Proszę użyć http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"zanim zmiany zostaną udostępnione lub spróbować później."

#: ../UpdateManager/Core/MyCache.py:318
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Lista zmian nie jest jeszcze dostępna.\n"
"\n"
"Należy użyć http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"do momentu, kiedy zmiany będą dostępne, lub spróbować później."

#: ../UpdateManager/Core/UpdateList.py:49
msgid "Failed to detect distribution"
msgstr "Nie udało się wykryć dystrybucji"

#: ../UpdateManager/Core/UpdateList.py:50
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Wystąpił błąd \\\"%s\\\" podczas sprawdzania używanego systemu."

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Important security updates"
msgstr "Ważne aktualizacje bezpieczeństwa"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Recommended updates"
msgstr "Aktualizacje polecane"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Proposed updates"
msgstr "Aktualizacje proponowane"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Backports"
msgstr "Backporty"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Distribution updates"
msgstr "Aktualizacje dystrybucji"

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr "Inne aktualizacje"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Należy sprawdzić dostępność aktualizacji samodzielnie</big></b>\n"
"\n"
"Aktualizacje automatyczne nie są włączone. Można ustawić tę funkcję w "
"<i>Źródłach oprogramowania</i> w zakładce <i>Aktualizacje</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr ""
"<big><b>Nie wszystkie aktualizacje mogą zostać zainstalowane</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Praca na zasilaniu z akumulatora</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Uruchamianie menedżera aktualizacji</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Zmiany"

#: ../data/gtkbuilder/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Sprawdź"

#: ../data/gtkbuilder/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "_Kontynuuj"

#: ../data/gtkbuilder/UpdateManager.ui.h:10
msgid "Description"
msgstr "Opis"

#: ../data/gtkbuilder/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Opis aktualizacji"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Bezpieczniej jest podłączyć komputer do zasilania przed dokonaniem "
"aktualizacji."

#: ../data/gtkbuilder/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Należy uruchomić częściową aktualizację, aby zainstalować jak najwięcej "
"aktualizacji. \n"
"\n"
"Może to być spowodowane przez:\n"
" * Poprzednio nieukończoną aktualizację\n"
" * Problemy z zainstalowanym oprogramowaniem\n"
" * Nieoficjalne pakiety, niedostarczone przez Ubuntu\n"
" * Normalne zmiany testowej wersji Ubuntu"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Postęp pobierania poszczególnych plików"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
msgid "Starting Update Manager"
msgstr "Uruchamianie menedżera aktualizacji"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
msgid "U_pgrade"
msgstr "_Aktualizuj"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Menedżer aktualizacji"

#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "You are connected via a wireless modem."
msgstr "Połączenie z Internetem nawiązane jest przez modem bezprzewodowy."

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Połączenie z Internetem jest nawiązane przez usługę roamingu. Aktualizacja "
"może okazać się kosztowna."

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Komputer jest zasilany z akumulatora. Na pewno kontynuować?"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "_Hide this information in the future"
msgstr "_Ukryj tę informację w przyszłości"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
msgid "_Install Updates"
msgstr "_Zainstaluj aktualizacje"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "_Partial Upgrade"
msgstr "_Częściowa aktualizacja"

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "_Settings..."
msgstr "U_stawienia..."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Upgrade"
msgstr "_Aktualizuj"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
msgid "updates"
msgstr "aktualizacje"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Nowa wersja Ubuntu jest dostępna. Dokonać aktualizacji?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Spytaj później"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Nie aktualizuj"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Tak, zaktualizuj teraz"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Można zaktualizować później otwierając Menedżer aktualizacji i klikając "
"przycisk \"Zaktualizuj\"."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Zrezygnowano z aktualizacji do nowej wersji Ubuntu"

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Pokaż i zainstaluj dostępne aktualizacje"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Aktualizacje oprogramowania"

#: ../update-manager:54 ../do-release-upgrade:41
msgid "Show version and exit"
msgstr "Wyświetl wersję i wyjdź"

#: ../update-manager:57
msgid "Directory that contains the data files"
msgstr "Katalog zawierający pliki danych"

#: ../update-manager:60
msgid "Check if a new Ubuntu release is available"
msgstr "Sprawdza, czy dostępne jest nowe wydanie Ubuntu"

#: ../update-manager:63 ../do-release-upgrade:44 ../check-new-release-gtk:176
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Sprawdź możliwość aktualizacji do najnowszej wersji testowej"

#: ../update-manager:67
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""
"Zaktualizuj, używając najnowszej, sugerowanej wersji aktualizatora wydań"

#: ../update-manager:74
msgid "Do not focus on map when starting"
msgstr "Nie skupiaj się na mapie podczas uruchamiania"

#: ../update-manager:77
msgid "Try to run a dist-upgrade"
msgstr "Spróbuj uruchomić aktualizację dystrybucji"

#: ../update-manager:80
msgid "Do not check for updates when starting"
msgstr "Nie wyszukuj aktualizacji podczas uruchamiania"

#: ../update-manager:84 ../do-release-upgrade:60
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Aktualizacja testowa z nakładką sandbox aufs"

#: ../update-manager:104
msgid "Running partial upgrade"
msgstr "Częściowa aktualizacja w toku"

#: ../do-release-upgrade:48 ../check-new-release-gtk:180
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Zaktualizuj do najnowszej wersji, używając aktualizatora z $distro-proposed"

#: ../do-release-upgrade:52
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Uruchom w specjalnym trybie aktualizacji.\n"
"Obecnie obsługiwane są tryby \"desktop\" dla komputerów biurkowych i "
"\"server\" dla aktualizacji serwerów."

#: ../do-release-upgrade:58
msgid "Run the specified frontend"
msgstr "Uruchom określoną nakładkę"

#: ../do-release-upgrade:63
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Sprawdź tylko, czy nowa wersja dystrybucji jest dostępna i przekaż rezultat "
"poprzez kod wyjścia"

#: ../do-release-upgrade:77
msgid "Checking for a new ubuntu release"
msgstr "Sprawdzanie dostępności nowego wydania Ubuntu"

#: ../do-release-upgrade:91
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Aby uzyskać informacje o aktualizacji, zajrzyj pod adres:\n"
"%(url)s\n"

#: ../do-release-upgrade:97
msgid "No new release found"
msgstr "Nie odnaleziono nowego wydania"

#: ../do-release-upgrade:109
#, c-format
msgid "New release '%s' available."
msgstr "Nowe wydanie '%s' jest dostępne"

#: ../do-release-upgrade:110
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Uruchom 'do-release-upgrade', aby zaktualizować."

#: ../check-new-release-gtk:91
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Aktualizacja do Ubuntu %(version)s jest dostępna"

#: ../check-new-release-gtk:132
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Zrezygnowano z aktualizacji do Ubuntu %s"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Niewprowadzona metoda: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Plik na dysku"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr "Pakiet .deb"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Zainstaluj brakujący pakiet."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Pakiet %s powinien zostać zainstalowany."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i zbędnych wpisów w pliku stanu"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Zbędne wpisy stanu w dpkg"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Zbędne wpisy stanu dpkg"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Jeśli pakiet kdelibs4-dev jest zainstalowany przed aktualizacją, to pakiet "
"kdelibs5-dev zostanie w jej trakcie zainstalowany. Szczegóły znajdują się w "
"raporcie błędu #279621 na stronie bugs.launchpad.net."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s musi zostać oznaczony jako zainstalowany ręcznie."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Proszę usunąć lilo, ponieważ grub także jest zainstalowany. (Więcej "
"szczegółów w raporcie błędu #314004.)"

#~ msgid ""
#~ "If this key is set the main update list window will show version information "
#~ "(from version to version)."
#~ msgstr ""
#~ "Jeśli ta opcja jest zaznaczona, główne okno z listą aktualizacji będzie "
#~ "wyświetlać informację o wersjach (źródłowa i docelowa)."

#~ msgid "Previous selected"
#~ msgstr "Poprzednio zaznaczone"

#~ msgid ""
#~ "This usually is caused by a system were /tmp is mounted noexec. Please "
#~ "remount without noexec and run the upgrade again."
#~ msgstr ""
#~ "Ta sytuacja jest zazwyczaj spowodowana przez system, gdy /tmp jest "
#~ "zamontowany z parametrem noexec. Zamontuj go ponownie bez parametru noexec i "
#~ "ponownie uruchom aktualizację."

#~ msgid "0 KB"
#~ msgstr "0 KB"

#~ msgid "The window size"
#~ msgstr "Rozmiar okna"

#~ msgid ""
#~ "Stores the state of the expander that contains the list of changes and the "
#~ "description"
#~ msgstr ""
#~ "Zapamiętuje położenie wysuwalnego obszaru zawierającego listę zmian i opis"

#~ msgid "Stores the size of the update-manager dialog"
#~ msgstr "Zapamiętuje rozmiar okna Menedżera aktualizacji"

#~ msgid "Remind to reload the channel list"
#~ msgstr "Przypominaj o odświeżaniu listy kanałów"

#~ msgid ""
#~ "If this key is set the install window will be automatically closed on "
#~ "successful installation."
#~ msgstr ""
#~ "Jeżeli to pole jest zaznaczone, okno instalacji zostanie automatycznie "
#~ "zamknięte po pomyślnym jej ukończeniu."

#~ msgid "*deprecated* Check for new distribution releases"
#~ msgstr "*niezalecane* Sprawdzaj nowe wydania dystrybucji"

#~ msgid "The package information was last updated less than one hour ago."
#~ msgstr ""
#~ "Informacje o pakietach zostały zaktualizowane mniej niż godzinę temu."

#~ msgid ""
#~ "It is unknown when the package information was updated last. Please try "
#~ "clicking on the 'Check' button to update the information."
#~ msgstr ""
#~ "Data ostatniej aktualizacji informacji o pakietach jest nieznana. Proszę "
#~ "zaktualizować informacje naciskając przycisk \"Sprawdź\"."

#~ msgid "_Check All"
#~ msgstr "Zaznacz wszystkie"

#~ msgid "_Uncheck All"
#~ msgstr "Odznacz wszystkie"

#~ msgid ""
#~ "This is most likely a bug in the upgrade tool. Please report it as a bug"
#~ msgstr ""
#~ "To najprawdopodobniej błąd w narzędziu aktualizacji. Proszę zgłosić to jako "
#~ "błąd."

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug."
#~ msgstr ""
#~ "W wyniku aktualizacji informacji o repozytoriach powstał błędny plik. Proszę "
#~ "zgłosić to jako błąd."

#~ msgid ""
#~ "It was impossible to install a required package. Please report this as a "
#~ "bug. "
#~ msgstr ""
#~ "Nie można było zainstalować wymaganego pakietu. Proszę zgłosić to jako błąd. "

#, python-format
#~ msgid "%.0f KB"
#~ msgstr "%.0f KB"

#~ msgid "1 KB"
#~ msgstr "1 KB"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration Menu later."
#~ msgstr ""
#~ "Poniższe aktualizacje zostały udostępnione po wydaniu Ubuntu. Można je "
#~ "zainstalować później, wybierając pozycję \"Menedżer aktualizacji\" z menu "
#~ "Administracja."

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this as a bug "
#~ "against the 'update-manager' package and include the files in /var/log/dist-"
#~ "upgrade/ in the bug report."
#~ msgstr ""
#~ "Przygotowanie systemu do aktualizacji nie powiodło się. Proszę zgłosić ten "
#~ "błąd pakietu \"update-manager\", w raporcie proszę zawrzeć pliki z "
#~ "/var/log/dist-upgrade/."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The upgrade "
#~ "will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug against the 'update-manager' package and include "
#~ "the files in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "System nie mógł uzyskać wstępnych wymagań do aktualizacji. Aktualizacja "
#~ "zostanie przerwana, pierwotny stan systemu zostanie przywrócony.\n"
#~ "\n"
#~ "Proszę zgłosić ten błąd pakietu \"update-manager\", w raporcie proszę "
#~ "zawrzeć pliki z /var/log/dist-upgrade/."

#, python-format
#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bug report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Proszę zgłosić ten błąd pakietu \"update-manager\", w raporcie proszę "
#~ "zawrzeć pliki z /var/log/dist-upgrade/.\n"
#~ "%s"

#, python-format
#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Po aktualizacji informacji o pakietach niezbędny pakiet \"%s\" nie może "
#~ "zostać odnaleziony.\n"
#~ "To wskazuje na poważny problem, proszę zgłosić błąd pakietu \"update-"
#~ "manager\" i zawrzeć w raporcie pliki z /var/log/dist-upgrade/."

#~ msgid ""
#~ "If none of this applies, then please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Jeśli żaden nie dotyczy, to proszę zgłosić błąd pakietu \"update-manager\" i "
#~ "zawrzeć w raporcie pliki z /var/log/dist-upgrade/."

#~ msgid "*DEPRECATED* this option will be ignore"
#~ msgstr "*PRZESTARZAŁA* - opcja zostanie zignorowana"

#, python-format
#~ msgid ""
#~ "Changes for the versions:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"
#~ msgstr ""
#~ "Zmiany w wersjach:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"

#~ msgid "This change is not coming from a source that supports changelogs."
#~ msgstr "Ta zmiana nie pochodzi ze źródła obsługującego dzienniki zmian."

#~ msgid "Settings..."
#~ msgstr "Ustawienia..."

#~ msgid "First run welcome message"
#~ msgstr "Komunikat powitalny po pierwszym uruchomieniu"

#~ msgid "Auto close the install window after successful install"
#~ msgstr "Automatycznie zamknij okno instalacji po jej pomyślnym zakończeniu"

#~ msgid "Show details of an update"
#~ msgstr "Pokaż szczegóły aktualizacji"

#~ msgid ""
#~ "If automatic checking for updates is disabled, you have to reload the "
#~ "channel list manually. This option allows to hide the reminder shown in this "
#~ "case."
#~ msgstr ""
#~ "Jeżeli automatyczne sprawdzanie aktualizacji jest wyłączone, odśwież listę "
#~ "kanałów ręcznie. Ta opcja pozwala na ukrycie przypominania o tym fakcie."

#~ msgid "If this key is set a first run welcome message will be presented."
#~ msgstr ""
#~ "Jeżeli ten klucz jest ustawiony, pierwszy komunikat powitalny zostanie "
#~ "wyświetlony."

#~ msgid "Show version in update list"
#~ msgstr "Wyświetl wersje na liście dostępnych aktualizacji"

#~ msgid ""
#~ "This key is deprecated in favor of the file /etc/update-manager/release-"
#~ "upgrades Check automatically if a new version of the current distribution is "
#~ "available and offer to upgrade (if possible)."
#~ msgstr ""
#~ "Ten klucz jest przestarzały i zostanie zastąpiony plikiem /etc/update-"
#~ "manager/release-upgrades. Automatycznie sprawdzaj, czy jest dostępne nowe "
#~ "wydanie bieżącej dystrybucji i zasugeruj aktualizację, jeżeli jest ona "
#~ "możliwa."

#~ msgid ""
#~ "The upgrade has aborted. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "Aktualizacja została przerwana. Proszę sprawdzić połączenie internetowe lub "
#~ "nośnik instalacyjny i sprawdzić ponownie. Wszystkie dotychczas pobrane pliki "
#~ "zostaną zachowane."

#~ msgid "Show package summary before package name in update list"
#~ msgstr ""
#~ "Wyświetlanie podsumowań pakietów przed nazwami pakietów na liście "
#~ "aktualizacji"

#~ msgid "make check-new-release-gtk ignore a given new release"
#~ msgstr "sprawia, że check-new-release-gtk ignoruje nowe wydania"

#~ msgid ""
#~ "This will permanently hide the new release prompt from check-new-release-"
#~ "gtk. Note that the small button in the main update-manager UI will still be "
#~ "there."
#~ msgstr ""
#~ "To spowoduje trwałe ukrycie zapytania o instalację nowej wersji systemu "
#~ "pochodzącej z check-new-release-gtk. Przycisk znajdujący się w głównym oknie "
#~ "menadżera aktualizacji pozostanie nadal widoczny."

#~ msgid ""
#~ "If this key is set, the main update list window will show the package "
#~ "summary before the package name."
#~ msgstr ""
#~ "Ustawienie tego klucza powoduje, że główne okna listy aktualizacji wyświetli "
#~ "podsumowania pakietów przed nazwami pakietów."

#~ msgid "<b><big>Upgrading Ubuntu to version 11.04</big></b>"
#~ msgstr "<b><big>Aktualizacja Ubuntu do wersji 11.04</big></b>"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from Applications "
#~ "later."
#~ msgstr ""
#~ "Następujące aktualizacje oprogramowania zostały stworzone po wydaniu tej "
#~ "wersji Ubuntu. Można zainstalować je później, uruchamiając Menedżera "
#~ "aktualizacji w menu System, jeśli teraz jest to niemożliwe."
